dallinger/__init__.py

```diff
@@ -1,20 +1,5 @@
 """This is Dallinger, a platform for simulating evolution with people."""
 
-from . import (
```

Comment:
I am a bit nervous about this code. There's no context explaining why it exists. It could have been done this way to solve a subtle import order problem. However I believe there should be better ways of avoiding such problems, no?

Reply:
I wonder how much things improve with _only this change_. Do you know, @pmcharrison?

Reply:
I tried it and didn't see much of an effect, I think because many of the modules are still imported elsewhere.